permissions/actions.go

```diff
@@ -20,6 +20,7 @@ const (
 	WriteObjectAction      = "fs:WriteObject"
 	DeleteObjectAction     = "fs:DeleteObject"
 	ListObjectsAction      = "fs:ListObjects"
+	DeleteObjectsAction    = "fs:DeleteObjects"
```

Comment:
It should not be a seperate action. Our authorization is modeled after S3, where deleting is an object level operation that requires a DeleteObject permission

Reply:
This is why the authorization currently happens in the Handle func - we parse and authorize every individual object to ensure the user has DeleteObject permissions for it

Reply:
Got it.